tests/parm/ccpp.gocart.nml.IN

```diff
@@ -272,7 +272,7 @@
        FNVETC   = "global_vegtype.igbp.t126.384.190.rg.grb",
        FNSOTC   = "global_soiltype.statsgo.t126.384.190.rg.grb",
        FNSMCC   = "global_soilmgldas.t126.384.190.grb",
-       FNMSKH   = "seaice_newland.grb",
+       FNMSKH   = "global_slmask.t1534.3072.1536.grb",
```

Comment:
Is this regular lat-lon or gaussian grid? If it's gaussian why?

Reply:
This is the Gaussian grid file we have so far. A high resolution lat/lon file can be used when available.